go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/aws/aws-sdk-go-v2/credentials v1.17.27
 	github.com/aws/aws-sdk-go-v2/service/s3 v1.58.2
 	github.com/cloudflare/cloudflare-go v0.100.0
-	github.com/cloudflare/cloudflare-go/v2 v2.4.0
```

Comment:
let's uncommit this to avoid breaking the reference in the config.

Reply:
Reverted in d82b934